Program.cs

```diff
@@ -61,31 +62,19 @@ static void Run(Args a)
         // it resides in <builddefname>/<guid>/VSTSDrop.json
         private static string ExtractDropUrl(string workingDirectory)
         {
-            //could take an envdir on what the build dir is for now though we just have one build
-            string buildDirectory = string.Empty;
             string guidDirectory = string.Empty;
 
             try
             {
-                buildDirectory = Directory.GetDirectories(workingDirectory).Single();
+                guidDirectory = Directory.GetDirectories(workingDirectory).Single();

```

Comment:
So yeah we should make it configurable or just recursively search for vstsdrop.json and use the first one it finds.

Reply:
that sounds reasonable